src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -548,16 +568,19 @@ def followstrand(f, factors, x0, x1, y0a, prec=53) -> list:
                 ci = c.imag()
                 coefsfactors += list(cr.endpoints())
                 coefsfactors += list(ci.endpoints())
-    from sage.libs.sirocco import contpath, contpath_mp, contpath_comps, contpath_mp_comps
+    from sage.libs.sirocco import contpath, contpath_mp, contpath_comps
+    from sage.libs.sirocco import contpath_mp_comps
```

Comment:
```suggestion
    from sage.libs.sirocco import (contpath, contpath_mp, contpath_comps
                                   contpath_mp_comps)
```
This is actually faster since it is only one import.